db/vita_partners.yml

```diff
@@ -451,3 +451,10 @@ vita_partners:
   source_parameters:
   - nccc
   logo_path: partner-logos/ncc-logo.png
+- name: La Casa de Esperanza (WI)
+  zendesk_instance_domain: eitc
+  zendesk_group_id: '360011477453'
+  display_name: La Casa de Esperanza (WI)
+  source_parameters:
+  - lce
```

Comment:
According to the Airtable record (https://airtable.com/tbltKS0IenBreuCXM/viwYWPLqTy5LrCrCa/recjMXheDjswA8YGD?blocks=hide) this should be `WILCDE`